sentry-android-replay/src/main/java/io/sentry/android/replay/util/DebugOverlayDrawable.kt

```diff
@@ -0,0 +1,66 @@
+package io.sentry.android.replay.util
+
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.ColorFilter
+import android.graphics.Paint
+import android.graphics.PixelFormat
+import android.graphics.Rect
+import android.graphics.drawable.Drawable
+
+internal class DebugOverlayDrawable : Drawable() {
+
+    private val paint = Paint(Paint.ANTI_ALIAS_FLAG)
+    private val padding = 6f
+    private val tmpRect = Rect()
+    private var masks: List<Rect> = emptyList()
+
+    override fun draw(canvas: Canvas) {
+        paint.textSize = 32f
+        paint.setColor(Color.BLACK)
+
+        paint.strokeWidth = 4f
+
+        for (mask in masks) {
+            paint.setColor(Color.BLACK)
+            paint.style = Paint.Style.STROKE
+            canvas.drawRect(mask, paint)
+
+            paint.style = Paint.Style.FILL
+            val label = "${mask.left} ${mask.top}"
```

Comment:
ideally I'd want to have bottom and right too, shall we maybe use a smaller textSize but in include all the bounds? e.g. in this format `0, 0 - 1, 1`